### PR TITLE
Dismiss Picker View controller on Done Button tap

### PR DESCRIPTION
**Pull request by aashishdhawan**
Whats new in the pull request?
one

the bottom right button has two states - Cancel and Done.

Cancel, when no image is selected.
Done, when some images are selected.

When you tap on Cancel button picker gets dismissed but this does not happen when you tap on Done button. This is not an expected behaviour. 

Although i am not sure if this behaviour was intentional.

### CODE REVIEW

**Comment by zenangst** [comment]
@aashishdhawan could you fix the merge conflicts?

**Comment by aashishdhawan** [comment]
@zenangst Done.

**Comment by zenangst** [comment]
@aashishdhawan 🚀

File: Source/ImagePickerController.swift [hunk 1]
```diff
@@ -328,6 +328,7 @@ extension ImagePickerController: BottomContainerViewDelegate {
   func doneButtonDidPress() {
     let images = ImagePicker.resolveAssets(stack.assets)
     delegate?.doneButtonDidPress(self, images: images)
+    dismissViewControllerAnimated(true, completion: nil)
```
Comment (SamreenShigri) [inline Source/ImagePickerController.swift]: Done Button isn't working, please give me solution

**Pull request merged**